### ISSUE TITLE
[GLES3 - Android] WorldEnvronment freezes the game

### ISSUE THREAD

**Issue by ArmonOyster** [godotengine/godot#42026]
**Godot version:**
Godot 3.2.2 stable

**OS/device including version:**
Vivo Y51
Android 5.0.2 (Lollipop), Funtouch 2.5
Qualcomm Adreno 306 - GLES3

**Issue description:**
The WorldEnvironment node with a new environment and has glow enabled results to freeze on the software on my android.

**Steps to reproduce:**
1. Set Renderer to GLES3
2. Add WorldEnvironment node
3. Create a new environment
4. Set Background->Mode to canvas
5. Enable glow in the new environment
6. Run in android

**Minimal reproduction project:**
[GLES3 Test for android.zip](https://github.com/godotengine/godot/files/5214259/GLES3.Test.for.android.zip)

**Comment by Calinou** [comment]
Does this bug occur when using the GLES2 renderer?

**Comment by ArmonOyster** [comment]
> Does this bug occur when using the GLES2 renderer?

No, and the glow effect on the WorldEnvironment node won't show on GLES2 renderer.
